### ISSUE TITLE
Printing from a module spec omits first few pages

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#1003]
When I print a Bible Module directly in PTXPrint the first few pages are skipped. When I load the same Module spec. into XXA and print that book in PTXPrint it prints correctly. (Ignore too-big titles and other bits, I need to get this working before fine-tuning it).

I've included the current PTXPrint "Scope" setting above each PDF in the pics to identify them. The first pic is correct. The second pic shows "page 1" starting with the third reading (Epistle) under "1st Sunday of Advent." 

1. I upgraded PTXPrint to the latest version today; the same thing happened with the previous version last week.
2. The same Module file is called for both printings. Same thing happens with all of my 10+ Catholic Sunday Readings modules.
3. I upgraded PTXPrint to the latest version today; the same thing happened with the previous version last week.
4. It is much more convenient to print directly from the Module specs, as I'm sure you know. I hope this isn't hard to fix.
5. Another odd thing I just noticed: The PDF with the first 3-4 pages missing has 62 total pages. The file that includes the entire module text has 61 pages! I haven't examined this yet but it looks as if styles might be applied differently, because they are both printed with the same PTXPrint Layout.

![Image](https://github.com/user-attachments/assets/ee17bbe1-a702-442e-b2e0-11bf164a1107)
![Image](https://github.com/user-attachments/assets/2908fdad-9890-40c3-8e08-b6127d758a22)

[kyxPublx-LiturgyReadingsPTXprintArchive.zip](https://github.com/user-attachments/files/17825854/kyxPublx-LiturgyReadingsPTXprintArchive.zip)

**Comment by davidg-sil** [comment]
Thanks for the archive, I can confirm it breaks... But not quite as described. As a module it appears as you include, but  in XXA for me the references are not getting decoded, and the first bit is chopped off:
![Image](https://github.com/user-attachments/assets/f6ebe0e3-68c1-4b88-9122-e6b671834bb2)

Hopefully MP or MH can solve the mystery!

**Comment by KimB2017** [comment]
I see that I did NOT yet have \v 5a and \v 5b entered in the Micah book for this project when I sent the archive yesterday. So I entered the verse segment markers, saved the PT project, then printed from PTXPrint again. The XXA output did not recognize my new verse segments: It still printed all of v. 5. (The module printed the whole chapter again.)

Back in PT I switched the XXA view to Standard Output, then reran the PDF. This time Micah 5:2-5a was correctly extracted. (Still no change in the module PDF.) I don't know if this was triggered by switching the XXA view from Specification to Output, or by doing a second PTXPrint run after changing the Paratext file. Could this be why davidg-sil got a different result in his XXA run from my archive? 

This also might help with https://support.bible/13232/verse-segments-seem-to-be-ignored-in-ptxprint. 

**Second error with Module printing**: In any case, printing directly from the Module spec caused the entire chapter of Micah 5 to be printed, whether or not the Scripture text correctly split the verse.

**Comment by KimB2017** [comment]
Yikes! I did not mean to "Close as completed"! I don't seem to have permission to Reopen the issue, but I don't consider it resolved. 

There are definitely some weird issues with printing a Module directly from PTX Print, vs. loading a Module into a Ptx XXx book and printing that book. I can set aside some time in mid-Dec or January for testing if it would help you figure this out, because in theory the direct method is much easier than using XXx books.

To summarize, there are at least two issues with direct Module printing in my sample here:
1. The first 3-4 pages of the file are omitted from the printout
2. When a \ref contains a verse segment the printout not only ignores the segments but also the end of the passage, printing the whole chapter.  (Whereas printing a module via an XXx book does respect verse segments.)

**Comment by davidg-sil** [comment]
I've reopened the issue. There's obviously a disagreement between PTXprint and paratext about how a module file should be read. That's sad, and no doubt Mark and Martin can provide some input when they've got a few spare synapses from the other deep work they're doing in other bits of the code....
